packages/ava-react/zh-CN/README.zh-CN.md

```diff
@@ -13,10 +13,18 @@ AVA 的 React 组件库
 
 </div>
 
-## 组件
-- NarrativeTextVis
+## 简介
 
-## 安装
+[@antv/ava-react](https://www.npmjs.com/package/@antv/ava-react) 是基于 AVA 能力整合而成的即插即用的 React 组件库， 它包含三大核心组件：
+
+* <span style="background-color:#A9A9A9; padding:2px 4px; border-radius:4px;color: black;">\<NarrativeTextVis \/></span>：展示数据洞察解读文本。在数据分析的全流程展示中，使用文本描述数据现象和给出洞察结论同样关键。
+* <span style="background-color:#A9A9A9; padding:2px 4px; border-radius:4px;color: black;">\<InsightCard \/></span>：以图文结合的方式展示数据洞察。既可以直接接收数据并自动进行洞察，也可以仅对洞察结果数据进行可视化和解读的展示。
+* <span style="background-color:#A9A9A9; padding:2px 4px; border-radius:4px;color: black;">\<AutoChart \/></span>>：根据数据自动推荐合适的图表并渲染。它可以为用户提供一行代码实现智能可视化的能力，目前该组件正在进一步升级中。
```

Comment:
Have you previewed these mds? It seems like there are some problems.
![image](https://github.com/antvis/AVA/assets/15177000/b976e498-3dcf-4d51-8e19-d4154c88e919)

Reply:
1. The left slider, which is not shown in the PREVIEW on github.
<img width="937" alt="image" src="https://github.com/antvis/AVA/assets/61140088/1dffaaf5-6e25-4b63-9ba9-cdedc8831e58">

2. The extra > symbol, which has been removed